src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp

```diff
@@ -26,9 +26,43 @@
 
 #include <map>
 
+#ifdef _WIN32
+#undef UNICODE
+#include <Shlobj.h>
+#include <Shlwapi.h>
+#include <Knownfolders.h>
+#elif defined(__APPLE__)
+#include <unistd.h>
+#endif
+
 namespace
 {
 
+std::string lithuanianPKCS11Path() {
+#ifdef _WIN32
+    PWSTR programFilesX86 = 0;
+    SHGetKnownFolderPath(FOLDERID_ProgramFilesX86, 0, nullptr, &programFilesX86);
```

Comment:
I suggest that we refactor _Program Files_ subpath logic into a separate folder, see [here](https://github.com/web-eid/libelectronic-id/pull/2/files#diff-1d1e9d13402083e3e4cfef76240e0afa5c7dbd2c004dbd19153acbe3703be28dR42-R52). But no need to change your code, this is just for your information. Let's do the refactoring when adding Croatian eID support to _devel_.